RAG_supporters/dataset/steering/steering_config.py

```diff
@@ -0,0 +1,170 @@
+"""Steering configuration with embedded steering modes."""
+
+import logging
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Dict, List, Optional, Tuple
+
+
+class SteeringMode(Enum):
+    """Steering embedding modes for cluster/subspace steering."""
+
+    SUGGESTION = "suggestion"  # Use suggestion embeddings
+    LLM_GENERATED = "llm_generated"  # LLM-generated steering text
+    CLUSTER_DESCRIPTOR = "cluster_descriptor"  # Cluster/topic descriptor embeddings
+    ZERO = "zero"  # Zero baseline (no steering)
+    MIXED = "mixed"  # Weighted combination of multiple modes
```

Comment:
Remove mixed as is not necessary now (can be done by list of picked other modes)